src/Filter/AbstractDateFilter.php

```diff
@@ -178,11 +178,21 @@ private function filterRange(ProxyQueryInterface $query, string $field, FilterDa
         $type = $data->getType() ?? DateRangeOperatorType::TYPE_BETWEEN;
 
         if (DateRangeOperatorType::TYPE_NOT_BETWEEN === $type) {
-            $this->applyType($query, $this->getOperator(DateOperatorType::TYPE_LESS_THAN), $field, $value['start']);
-            $this->applyType($query, $this->getOperator(DateOperatorType::TYPE_GREATER_THAN), $field, $value['end']);
+            if ($value['start']) {
```

Comment:
shouldn't php-strict-rules complain about `$value['start']` not being a `boolean`? 🤔